components/org.wso2.carbon.identity.oauth.endpoint/src/main/java/org/wso2/carbon/identity/oauth/endpoint/authz/OAuth2AuthzEndpoint.java

```diff
@@ -2361,7 +2361,8 @@ private String doUserAuthorization(OAuthMessage oAuthMessage, String sessionData
             return promptUserForConsent(sessionDataKeyFromLogin, oauth2Params, authenticatedUser, true, oAuthMessage);
         } else if (isPromptNone(oauth2Params)) {
             return handlePromptNone(oAuthMessage, sessionState, oauth2Params, authenticatedUser, hasUserApproved);
-        } else if (isPromptLogin(oauth2Params) || isPromptParamsNotPresent(oauth2Params)) {
+        } else if (isPromptLogin(oauth2Params) || isPromptParamsNotPresent(oauth2Params)
+                || isPromptSelectAccount(oauth2Params)) {
```

Comment:
Just for reference, if possible let's add a comment saying why prompt_select is handled in this manner